### DISCUSSION TITLE
遍历子图节点报错

### DISCUSSION THREAD

**Discussion by zjarles** [alibaba/GraphScope#3225] [Q&A]
将子图转换为networks图后，在循环遍历时，报错如下，请问是什么原因呢？

G = nx.DiGraph(sub_graph, default_label="trmnl")  
for node in G.nodes(data=True):      
    print(node)

AnalyticalEngineInternalError: /work/analytical_engine/core/object/object_manager.h:80: GetObject -> Object 
graph_GWXIIwo7 does not exist
boost::leaf::result<std::shared_ptr<gs::IFragmentWrapper> > 
gs::ObjectManager::GetObject<gs::IFragmentWrapper>(std::string const&) + 0x2FE
gs::GrapeI ... [truncated]

这个图是存在的，sub_graph.schema可以显示数据。

**Comment by acezen** [comment]
Thanks for the report, can't find the reason from the log. I will try to reproduce the problem.
